helmfile/upstream/harbor/values.yaml

```diff
@@ -284,7 +329,7 @@ persistence:
       #storageclass: STANDARD
       #multipartcopychunksize: "33554432"
       #multipartcopymaxconcurrency: 100
-      multipartcopythresholdsize: "33554432"
+      #multipartcopythresholdsize: "33554432"
```

Comment:
I believe this shouldn't be commented out

see this commit: https://github.com/elastisys/compliantkubernetes-apps/commit/e151dfc885b44cf1e89880fd79ae26dcf8cdac2e